garak/generators/rest.py

```diff
@@ -193,6 +203,7 @@ def _call_model(
             data_kw: request_data,
             "headers": request_headers,
             "timeout": self.request_timeout,
+            "proxies": self.proxies,
         }
```

Comment:
If passing `None` for the `proxies` value suppress the env variable lookups by `requests` then this should defer to only set if passed in config.
```suggestion
        }
        if hasattr(self, "proxies") and self.proxies is not None:
            req_kArgs["proxies"] = self.proxies
```